md/migrate-from-an-earlier-version-of-bonita-bpm.md

```diff
@@ -373,10 +373,18 @@ Bonita 7.11+ supports Oracle 19c version. To migrate to Bonita 7.11+ when using
 please follow this procedure:
 * ensure your Bonita platform is shut down
 * in a first step, run Bonita migration tool to update Bonita platform to version 7.10.5, following the procedure above
-* then upgrade your Oracle database server to the version 19c, version 7.10.x is compatible with 12c and 19c. 
+* then upgrade your Oracle database server to the version 19c, version 7.11.x is compatible with 12c and 19c. 
```

Comment:
```suggestion
* then upgrade your Oracle database server to version 19c (version 7.10.x is compatible with 12c and 19c.)
```
I did not catch this one. 7.11.x == 19c and 7.10.x == 19c + 12c

Reply:
I think both 7.11 and 7.10 are compatible with both database versions 19.c and 12.c
https://customer.bonitasoft.com/support-policies/2219

Reply:
I checked that, 
* 7.19 support 12.2 only
* 7.10 support 12.2 and 19
* 7.11 support 19 only

![Screenshot 2020-12-07 at 17 01 53](https://user-images.githubusercontent.com/2739234/101374258-3afc2480-38ae-11eb-84e1-35f5afd1cb85.png)

Reply:
oh yes... my bad